index.html

```diff
@@ -468,31 +468,26 @@
 				// remove prior icons - otherwise it'll duplicate
 				$('#icons').remove();
 				// if information is unavailble this prevents issues with jquery
-				if (data.hasOwnProperty("network") && data.network.hasOwnProperty("mode"))
+				if (data.hasOwnProperty("network"))
```

Comment:
This check isn't needed - as we have already set `firmwareData.network` in all cases above.